enhancements/network/ovn-hypershift-live-migration.md

```diff
@@ -0,0 +1,295 @@
+---
+title: OVN-kubernetes live migration for hypershift kubevirt provider
+authors:
+  - @ellorent
+reviewers:
+  - "@dcbw"
+  - @danwinship
+  - @tssurya
+  - @numans
+approvers:
+  - TBD
+creation-date: 2023-02-06
+last-updated: 2023-02-06
+tracking-link: 
+  - "https://issues.redhat.com/browse/CNV-22946"
+see-also:
+  - "https://hypershift-docs.netlify.app/"
+---
+
+
+# OVN-kubernetes live migration for hypershift kubevirt provider
+
+## Summary
+
+The [Hypershift](https://hypershift-docs.netlify.app/) project creates openshift clusters on demand on top of already provisioned openshift by deploying "Hosted Control Planes" and attach worker nodes to "Hosted Control Planes" using oneof different providers, such as aws, azure, baremetal (agent), and kubevirt, in the case of kubevirt it will spins up the workers as kubevirt virtual machines.
+
+There are some network requirements, some of them are imposed by kubevirt 
+live migration and others by kubernetes:
+
+- Consistent IP follows VM during Live Migration, and is the IP of the primary guest node interface that the kubelet uses to register/communicate with.
+- Guest Cluster Network isolation. Guest clusters shouldn’t be able to talk to other guest clusters directly except through public LBs. Guest clusters shouldn’t be able to talk to any infra components except through public LBs
+- No TCP connection breakage over E/W and N/S during live migration with minimal disruption.
+- Support exposing workers with services 
+- Allow workers services acess to services like kubedns and apiserver from hosted control plane.
```

Comment:
```suggestion
- Allow workers services access to services like kubedns and apiserver from hosted control plane.
```

I'm confused; what do you mean by worker's services ? Maybe throw in a couple examples, to make it more explicit.